.github/scripts/build-images.sh

```diff
@@ -3,6 +3,18 @@
 # @authors: Somya Arora, Arash Pourhabibi
 # @modified: Shanqing Lin
 
+# 0. Setup QEMU and Docker buildx.
```

Comment:
This should be done inside the `if` and only when there is actually an image to build.